### PR TITLE
{lang}[fosscuda/2019a] R v3.6.0

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)
This is a build of R using the fosscuda toolchain. It has the same R packages as `R/3.6.0-foss-2019a` (#9184) with 3 additional packages that are specific for GPUs. Namely `OpenCL`, `RViennaCL` and `gpuR` plus their dependencies.

### CODE REVIEW

File: easybuild/easyconfigs/r/R/R-3.6.0-fosscuda-2019a.eb [hunk 1]
```diff
@@ -0,0 +1,2335 @@
...
+    ('libsndfile', '1.0.28'),  # for seewave
+    ('ICU', '64.2'),  # for rJava & gdsfmt
+    ('HDF5', '1.10.5'),  # for hdf5r
+    ('UDUNITS', '2.2.26'),  # for units
+    ('GSL', '2.5'),  # for RcppGSL
+    ('ImageMagick', '7.0.8-46'),  # for animation
+    # OS dependency should be preferred if the os version is more recent then
+    # this version, it's nice to have an up to date openssl for security
+    # reasons
+    # ('OpenSSL', '1.0.2h'),
```
Comment (Micket) [inline easybuild/easyconfigs/r/R/R-3.6.0-fosscuda-2019a.eb]: Should be version `1.1.1b`

Comment (lexming) [reply easybuild/easyconfigs/r/R/R-3.6.0-fosscuda-2019a.eb]: Fixed :+1:

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node154.hydra.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/08103e29b9c8494f5e29609f5b8ab8aa for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 7 out of 8 (1 easyconfigs in this PR)
node3302.joltik.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6242 CPU @ 2.80GHz, Python 2.7.5
See https://gist.github.com/185ba759e4393673433edcb91755ff03 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3301.joltik.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6242 CPU @ 2.80GHz, Python 2.7.5
See https://gist.github.com/26df03a44ea03b5931e02a2bc29db6d7 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @lexming!

**Pull request merged**